pkg/scheduler/framework/framework.go

```diff
@@ -704,18 +711,16 @@ func (f *framework) updatePolicySnapshotStatusFromBindings(
 	// Compare the new decisions + condition with the old ones.
 	currentDecisions := policy.Status.ClusterDecisions
 	currentCondition := meta.FindStatusCondition(policy.Status.Conditions, string(placementv1beta1.PolicySnapshotScheduled))
-	if equalDecisions(currentDecisions, newDecisions) && condition.EqualCondition(currentCondition, &newCondition) {
+	if observedCRPGeneration == policy.Status.ObservedCRPGeneration &&
```

Comment:
when will we hit the case that the decision/condition doesn't change but the CRP's generation has changed (selector changed)